signing/secp256k1/point.go

```diff
@@ -0,0 +1,135 @@
+package secp256k1
+
+import (
+	"github.com/ElrondNetwork/elrond-go-core/core/check"
+	crypto "github.com/ElrondNetwork/elrond-go-crypto"
+	"github.com/btcsuite/btcd/btcec"
+)
+
+var _ crypto.Point = (*secp256k1Point)(nil)
+
+type secp256k1Point struct {
+	Point *btcec.PublicKey
```

Comment:
can we have the btcec.PublicKey simply embedded here?
just like for ed25519

Reply:
done